### PR TITLE
Adding support to system channel docs 

### LINKED ISSUE #5929: [Docs] Adding support to ```system channel```
#### Current State
Currently the [docs](https://docs.meshery.io/reference/mesheryctl/system/channel) for the ```system channel``` command is outdated.

#### Desired State
Docs for ```system channel``` should be updated, you can take look at [system context](https://docs.meshery.io/reference/mesheryctl/system/context) docs, to get a overview of what changes are required. 

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by kamalbuilds**
Signed-off-by: Kamal Nayan <95926324+legendarykamal@users.noreply.github.com>

**Description**
Made system channel docs more descriptive as for a new user the commands are not familiar and how to use the commands to set,view channel .
This PR fixes #5929 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by alphaX86 — COMMENTED** [review]
LGTM for now...

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: mesheryctl/internal/cli/root/system/channel.go [hunk 1]
```diff
@@ -223,6 +223,12 @@ var channelCmd = &cobra.Command{
 	Example: `
 // Subscribe to release channel or version
 mesheryctl system channel [stable|stable-version|edge|edge-version]
```
Comment (nishantwrp) [inline mesheryctl/internal/cli/root/system/channel.go:225]: We can probably update this documentation too. I don't think `mesheryctl system channel` itself accepts any args.

Comment (alphaX86) [reply mesheryctl/internal/cli/root/system/channel.go:225]:
```suggestion
// Subscribe to release channel or version
mesheryctl system channel 
```

Comment (alphaX86) [reply mesheryctl/internal/cli/root/system/channel.go:225]: // @legendarykamal ⬆️

Comment (legendarykamal) [reply mesheryctl/internal/cli/root/system/channel.go:225]: ok yes @nishantwrp I agree it doesn't accept on itself , ok @alphaX86 I will update it now.

File: mesheryctl/internal/cli/root/system/channel.go [hunk 2]
```diff
@@ -223,6 +223,12 @@ var channelCmd = &cobra.Command{
...
+// To set the channel
+mesheryctl system channel set [stable|stable-version|edge|edge-version]
```
[the remark this answers is not in the document]
Comment (legendarykamal) [reply mesheryctl/internal/cli/root/system/channel.go:227]: @leecalcote I think there are only three commands set,view, switch which I have included.
Pin happens automatically when user sets a channel . 
and user gets this if trying to use pin cmd 
```
Error: unknown command "pin" for "mesheryctl system channel"
```

[the remark this answers is not in the document]
Comment (alphaX86) [reply mesheryctl/internal/cli/root/system/channel.go:227]: Whoa whoa @legendarykamal there isn't a command called `pin` 😅 I think you misunderstood. Lee meant that if you could pin/set a specific version of Meshery by typing like `mesheryctl system channel v0.6.0` like that...

[the remark this answers is not in the document]
Comment (legendarykamal) [reply mesheryctl/internal/cli/root/system/channel.go:227]: haha ok ok gotcha , thanks for clearing and ya i know  this I will fix it right away :)

**Comment by legendarykamal** [comment]
yes we are ready to go :100:

File: mesheryctl/internal/cli/root/system/channel.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (legendarykamal) [reply mesheryctl/internal/cli/root/system/channel.go:227]: @alphaX86 @leecalcote is one example enough or i should include one for like ```edge-version``` also?

File: mesheryctl/internal/cli/root/system/channel.go [hunk 3]
```diff
@@ -222,7 +222,15 @@ var channelCmd = &cobra.Command{
 	Args:  cobra.NoArgs,
 	Example: `
 // Subscribe to release channel or version
-mesheryctl system channel [stable|stable-version|edge|edge-version]
+mesheryctl system channel 
+// To set the channel
+mesheryctl system channel set [stable|stable-version|edge|edge-version]
+// To pin/set the channel to a specific version
+mesheryctl system channel set stable-version 0.6.0
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/system/channel.go:229]: Not apt :|

```suggestion
mesheryctl system channel set stable-v0.6.0-rc.6fc
```
This is the apt one

**Review by alphaX86 — CHANGES_REQUESTED** [review]
> yes we are ready to go 💯

Not yet... do the change requested above

File: mesheryctl/internal/cli/root/system/channel.go (hunk 3 above)
Comment (legendarykamal) [reply mesheryctl/internal/cli/root/system/channel.go:229]: sorry @alphaX86 I had not considered your example but an earler mesheryctl error which had redirected me to set the v as v0.6.0 so I had that in mind , 

Though I will make the rc one for now as you mentioned :+1:

**Comment by legendarykamal** [comment]
I confirmed again and am adding this for future reference 
image

**Comment by legendarykamal** [comment]
Is this good to merge now @alphaX86 @leecalcote ??

**Comment by nishantwrp** [comment]
@harkiratsm can you review this? Is it what you wanted the docs to be like? I'm asking because, in the issue you mentioned that the docs should be made similar to that of `mesheryctl system context` command, but in this pr it's done differently.

**Comment by harkiratsm** [comment]
@legendarykamal you were just required to make change on [docs](https://docs.meshery.io/reference/mesheryctl/system/context)  page, I have already a ongoing [pr](https://github.com/meshery/meshery/pull/5661) in which I had already solved the issue on mesheryctl. So you just required to make changes on docs page for [system context.](https://docs.meshery.io/reference/mesheryctl/system/context) .
If you are still having trouble ping me on slack.

**Comment by alphaX86** [comment]
> @legendarykamal you were just required to make change on [docs](https://docs.meshery.io/reference/mesheryctl/system/context)  page, I have already a ongoing [pr](https://github.com/meshery/meshery/pull/5661) in which I had already solved the issue on mesheryctl. So you just required to make changes on docs page for [system context.](https://docs.meshery.io/reference/mesheryctl/system/context) .
> If you are still having trouble ping me on slack.

@harkiratsm this change has to be made on golang one coz if consequent change has been made after this, the CI action will revert to original change (thus removing any additional change made in this PR) so I don't think, this should be made on docs page but rather on channel.go file

**Comment by harkiratsm** [comment]
Closing this PR, as this wasn't required for #5929 and this  [pr](https://github.com/meshery/meshery/pull/5661) fix for mesheryctl side.

**Comment by alphaX86** [comment]
Its just a minor change... Need to review and merge this quick..

**Comment by harkiratsm** [comment]
I had a call @alphaX86 he told me about the [mesheryctl workflow](https://github.com/meshery/meshery/blob/master/.github/workflows/mesheryctl-ci.yml) , which I wasn't aware about , So reopening the pr .

**Review by alphaX86 — DISMISSED** [review]
LGTM

**Review by harkiratsm — COMMENTED** [review]
LGTM :rocket:

**Comment by legendarykamal** [comment]
> I had a call with @alphaX86 he told me about the [mesheryctl workflow](https://github.com/meshery/meshery/blob/master/.github/workflows/mesheryctl-ci.yml) , which I wasn't aware about , So reopening the pr .

Ouch ! In the first PR i had made only in docs but yeah @alphaX86 guided me about mesheryctl flow :open_mouth:

**Comment by legendarykamal** [comment]
@alphaX86 @leecalcote If this looks good and aprooved can we get this merged now?

**Review by AugustasV — APPROVED** [review]
Looks good to me. @leecalcote it could be merged after changes approved.

File: mesheryctl/internal/cli/root/system/channel.go [hunk 4]
```diff
@@ -222,7 +222,15 @@ var channelCmd = &cobra.Command{
 	Args:  cobra.NoArgs,
 	Example: `
 // Subscribe to release channel or version
-mesheryctl system channel [stable|stable-version|edge|edge-version]
+mesheryctl system channel 
+// To set the channel
+mesheryctl system channel set [stable|stable-version|edge|edge-version]
+// To pin/set the channel to a specific version
+mesheryctl system channel set stable-v0.6.0-rc.6fc
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/channel.go:229]:
```suggestion
mesheryctl system channel set stable-v0.6.0
```

**Review by leecalcote — APPROVED** [review]

**Comment by legendarykamal** [comment]
Thank you @leecalcote for merging this one :rocket:

**Pull request merged**